Source/eepromcontrol.h

```diff
@@ -16,6 +17,20 @@
 #define ADDR_FLASH_SECTOR_9     ((uint32_t)0x080A0000) /* Base @ of Sector 9, 128 Kbyte */
 #define ADDR_FLASH_SECTOR_10    ((uint32_t)0x080C0000) /* Base @ of Sector 10, 128 Kbyte */
 #define ADDR_FLASH_SECTOR_11    ((uint32_t)0x080E0000) /* Base @ of Sector 11, 128 Kbyte */
+#else
+#define ADDR_FLASH_SECTOR_0     ((uint32_t)0x08000000) /* Base @ of Sector 0, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_1     ((uint32_t)0x08020000) /* Base @ of Sector 1, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_2     ((uint32_t)0x08040000) /* Base @ of Sector 2, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_3     ((uint32_t)0x08080000) /* Base @ of Sector 3, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_4     ((uint32_t)0x080a0000) /* Base @ of Sector 4, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_5     ((uint32_t)0x080c0000) /* Base @ of Sector 5, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_6     ((uint32_t)0x080e0000) /* Base @ of Sector 6, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_7     ((uint32_t)0x08100000) /* Base @ of Sector 7, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_8     ((uint32_t)0x0812000) /* Base @ of Sector 8, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_9     ((uint32_t)0x08140000) /* Base @ of Sector 9, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_10    ((uint32_t)0x08160000) /* Base @ of Sector 10, 128 Kbyte */
+#define ADDR_FLASH_SECTOR_11    ((uint32_t)0x08180000) /* Base @ of Sector 11, 128 Kbyte */
+#endif
```

Comment:
might as well put in the rest of the sectors. Or get rid of them. FlashStorage.cpp seems to assume all sectors are EEPROM_PAGE_SIZE long, so the address or sector can be calculated from EEPROM_PAGE_BEGIN.